reprex/doi.md

```diff
@@ -0,0 +1,66 @@
+``` r
+library(biblids)
+# this is the hard way to enter
+doi(
+  prefix = c("10.1038", "10.1000", "10.1007"),
+  suffix = c("nphys1170", NA, "978-3-642-65840-2_5")
+)
+#> <digital object identifier[3]>
+#> [1] 10.1038/nphys1170          
+#> [2] <NA>                       
+#> [3] 10.1007/978-3-642-65840-2_5
+```
+
+There's a `knit.print()` method, too:
+
+-   [`https://doi.org/10.1038/nphys1170`](https://doi.org/10.1038/nphys1170)
+-   `NA`
+-   [`https://doi.org/10.1007/978-3-642-65840-2_5`](https://doi.org/10.1007/978-3-642-65840-2_5)
+
+``` r
+# much easier to enter this way
```

Comment:
Interleaving reprexes are more difficult to run locally, should we repeat the `library()` call here?

```suggestion
library(biblids)
# much easier to enter this way
```

Reply:
yes, thanks for catching that!